YouTube.js

```diff
@@ -166,22 +170,40 @@ var testCases = [
 		"items": [
 			{
 				"itemType": "videoRecording",
-				"title": "HD Charlie Chaplin - The Great Dictator: Speech to Humanity | HD",
+				"title": "The Great Dictator - Speech to Humanity | HD",
 				"creators": [
 					{
-						"lastName": "EMS co",
-						"creatorType": "contributor",
-						"fieldMode": 1
+						"firstName": "E. M. S.",
+						"lastName": "co",
+						"creatorType": "contributor"
```

Comment:
I'd vote for treating all author names as institutional authors without attempting to parse them. I'm not sure how often  people actually use their full names on YouTube